test/unit_test/character_sheet_unit_tests/RaceUnitTests.java

```diff
@@ -2,26 +2,71 @@
 
 import static org.junit.Assert.*;
 
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
 import rpg_database.character_sheet.Background;
 import rpg_database.character_sheet.CharacterSheet;
 import rpg_database.character_sheet.Fields;
 import rpg_database.character_sheet.Race;
+import rpg_database.character_sheet.exceptions.CharacterSheetException;
 
 public class RaceUnitTests {
 
+	@Rule
+	public ExpectedException thrown = ExpectedException.none();
+
+	@Test
+	public void expectException_TestBackgroundHasMoreRaceOptionChoiceInvalidRace() {
+		expectExceptionWithMessage(CharacterSheetException.class, "Seheron Convert does not offer Qunari race for choicing!");

```

Comment:
A messaget meg tudjuk változtatni arra, hogy "Seheron Convert can not originate from Qunari heritage!" ?
Illetve ha paraméterek alapján generál error messaget a logikád, akkor írnál még egy tesztet? Csak hogy a message jól generálódik -e. (mert ugye ennyi erővel ez a hardcodeolt string is lehetne mögötte)

Reply:
ok